WordPressMocks/src/main/assets/mocks/mappings/wpcom/me/rest_v11_me_signup.json

```
@@ -2,7 +2,7 @@
     "scenarioName": "auth",
     "requiredScenarioState": "signed-up",
     "request": {
-        "urlPath": "/rest/v1.1/me/",
+        "urlPattern": "/rest/v1.1/me($|\\?.*|\/(?!settings|sites))",
```

Same as above, this should do the same thing:

```suggestion
        "urlPattern": "/rest/v1.1/me(/)?($|\\?.*)",
```